### PR TITLE
Add content and tests for exercises

### PR DESCRIPTION
**Pull request by thomastuts**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,16 +7,50 @@ JavaScript Reduce Lab
...
+];
+```
+
+Our output would look like this:
+
+```
+{ '7': 2, '9': 1 }
+```
+
+This output can be interpreted as: _'We have two sentences with 7 words, and one sentence with 9 words'_. That should
```
Comment (pletcher) [inline README.md:50]: Where does the nine-word sentence come from?

**Comment by pletcher** [comment]
This is great! I'm just a bit confused by the one example -- maybe I'm reading something wrong?

**Comment by thomastuts** [comment]
Sorry, problem on my end — forgot to add the last line. Fixed!

**Comment by pletcher** [comment]
LGTM!

**Pull request merged**